Sources/ArcGISToolkit/Components/OfflineMapAreasView/OfflineMapAreasView.swift

```diff
@@ -56,12 +62,20 @@ public struct OfflineMapAreasView: View {
             }
             .task {
                 await mapViewModel.makePreplannedOfflineMapModels()
+                mapViewModel.loadPreplannedMobileMapPackages()
             }
             .toolbar {
                 ToolbarItem(placement: .confirmationAction) {
                     Button("Done") { dismiss() }
                 }
             }
+            .onAppear {
+                UNUserNotificationCenter.current().requestAuthorization(options: [.alert, .sound]) { _, error in
+                    if let error {
+                        print(error.localizedDescription)
+                    }
+                }
```

Comment:
In my opinion eventually we can discard the error and only test the boolean state. If permission is granted, execute the send notification logic.